Sources/Controls/UIScrollView+Combine.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  UIScrollView+Combine.swift
+//  CombineCocoa
+//
+//  Created by Joan Disho on 09.08.19.
+//
+
+import Foundation
+import Combine
+
+public extension UIScrollView {
+    /// A publisher emitting content offset changes from this UIScrollView.
+    var contentOffsetPublisher: AnyPublisher<CGPoint, Never> {
+        publisher(for: \.contentOffset)
```

Comment:
This is just using KVO - Is that enough in this case?

Reply:
What do you mean? Why shouldn't be enough? 🤔

Reply:
Should be fine, just resolve the other things :) I'd also want you to add the usage to the Example project. a UITextView inherits from UIScrollView so you could use the output text view itself to demonstrate the usage of these two publishers.